### ISSUE TITLE
(minibug) Incorrect template for Warning message

### ISSUE THREAD

**Issue by lobster2012-user** [NLog/NLog#3190]
/NLog/LayoutRenderers/VariableLayoutRenderer.cs
```csharp
InternalLogger.Warn("${{var={0}}} should be declared as <variable name=\"var_{0}\" value=\"...\" /> and used like this ${var_{{0}}}. Because of unsafe Layout={1}", Name, layout);
```
must be
```
${{var_{{0}}}}
```

**Comment by snakefoot** [comment]
Thank you for reporting. Have created #3191

**Comment by 304NotModified** [comment]
Thanks @lobster2012-user! This will be fixed in NLog 4.6.1 👍
